specification/azurearcdata/resource-manager/Microsoft.AzureArcData/stable/2021-11-01/examples/CreateOrUpdateDataController.json

```diff
@@ -55,6 +63,14 @@
           "basicLoginInformation": {
             "username": "username"
           },
+          "metricsDashboardCredential": {
+            "username": "username",
+            "password": "********"
+          },
+          "logsDashboardCredential": {
+            "username": "username",
+            "password": "********"
```

Comment:
Should password actually be in the PUT response? Since its not normally part of the GET response. I'm not clear on what is best here, but it mainly seems like an additional disclosure risk to echo it here if its not required.